PureLayout/PureLayout/ALView+PureLayout.h

```diff
@@ -70,6 +70,29 @@ PL__ASSUME_NONNULL_BEGIN
 #endif /* PL__PureLayout_MinBaseSDK_iOS_8_0 */
 
 
+#pragma mark Pin Edges to SafeArea
+
+#if PL__PureLayout_MinBaseSDK_iOS_8_0
```

Comment:
Agreed. @smileyborg went through great lengths during various iOS versions to keep library upgrades compatible with newer versions of iOS SDKs. I think this stability has helped with PL’a adoption over time.

Reply:
We may need to use the 8_0 compatible API's to match MinBaseSDK_8_0. And when 9_0 SDK available, can make use of the new API. We should also consider a lot on backward compatibility first. safeAreaLayoutGuide only available on iOS 11, and only available when build with the iOS 11 SDK. 
But in runtime, it may run on iOS version < 11, such as iOS 8.0 (oldest target avaiable in the latest iOS 11 SDK & Xcode 9.2). While some developers still need to support the older iOS version such as iOS 8.0 using older version Xcode (for exp. 8.0). Using those preprocessor is important to cover up all the test case and not fail the testing.
I'll try to make some change to pass the testing on my fork after merging from @himaratsu.

Reply:
Hello, I've make a pull request to @himaratsu 's branch. Pls check.
Added some condition compilation defines and runtime conditions to avoid runtime crash in older version (iOS 8.0 etc.)